### ISSUE TITLE
Improve document

### ISSUE THREAD

**Issue by ryota-sakamoto** [awslabs/dynein#136]
## Background

For now, README.md is big and a lot of informations within one place. I think it is hard to maintain so we need to make README.md simple like other tools and specific informations should be written in specific page like how to use the command.
Also all of samples of command is maintained manually and it will cause differences from actual command.

## Work

1. make README.md simple, divide informations to specific page
2. use trycmd for all of command regard dynein in the docs

## Design

The doc `docs/cmd/*.md` is to describe each command how to use command and samples. 

```
.
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
├── Cargo.lock
├── Cargo.toml
├── LICENSE
├── NOTICE
├── README.md
├── docs
│   ├── cmd
│   │   ├── admin.md
│   │   └── use.md
│   ├── development.md
│   ├── faq.md
│   └── troubleshooting.md
├── k8s-deploy-dynamodb-local.yml
├── rust-toolchain
├── src
└── tests
```

## Ref

https://github.com/aws/aws-cli
https://github.com/aws/aws-sam-cli
https://github.com/aws/aws-nitro-enclaves-cli
https://github.com/aws/copilot-cli

**Comment by StoneDot** [comment]
I agree that splitting the documentation is necessary. However, I have concerns about the potential impact on user experience, especially for new users. The current style of `README.md` provides a straightforward guide that is easy for newcomers to follow.

Considering this, I would like to suggest a step-by-step approach. First, let's create book-style documentation using mdBook specifically tailored for new users. This will provide them with a comprehensive and structured resource, enhancing their understanding and overall experience. Once we have the new user documentation in place, we can proceed with splitting the `README.md`. By following this approach, we can ensure that new users have a reliable and easy-to-understand resource, while still maintaining the benefits of a divided documentation structure.
